services/frontend/pkg/config/defaults/defaultconfig.go

```diff
@@ -89,6 +89,16 @@ func DefaultConfig() *config.Config {
 			SupportedTypes:      []string{"sha1", "md5", "adler32"},
 			PreferredUploadType: "sha1",
 		},
+		ReadyOnlyAttributes: config.ReadyOnlyAttributes{
```

Comment:
Just for my understanding, no clever way of finding out if the built in libre idm is used and in that case having all these attributes writable automatically? 😢

Reply:
Unresolved the conversation, not convinced, yet. sorry.

oCIS single binary comes with a builtin IdP where the attributes are in fact writable. From an admin user perspective it's a huge step backwards if they now need to set environment variables to get the shipped artifact (ocis single binary) into it's "default state" of having writable fields.

For external IdPs it's of course a matter of correct configuration whether or not the fields are writable. But for the single binary that we ship you would cripple functionality.

Reply:
@rhafer What's your thoughts on this?

Reply:
I also want to join the party :smile: 

Wouldn't `ReadOnlyAttributes` anyways be empty in the single binary case? That would mean all a writable or not?

Reply:
Yes, that's basically what @rhafer wrote in his PR review. I agree, empty string as default for single binary would be correct and would solve my concerns.

If an external IdP is used it's most likely that the env variable must be filled and if forgotten the UI suggests wrong behaviour. But that's still better than setting most fields to read only and having `standard single binary` users confused. Happy to go with that solution.

Reply:
> @rhafer What's your thoughts on this?

That's why we should have and empty default here. So I guess this discussion is resolved, right?

> Wouldn't ReadOnlyAttributes anyways be empty in the single binary case? That would mean all are writable or not?

Yes. The setting should be empty by default. See also the other conversation, below.